debian/changelog

```diff
@@ -0,0 +1,270 @@
+z3 (4.8.10-1) unstable; urgency=medium
+
+  * New upstream release.
+  * Update Standards-Version to 4.5.1.
+  * Update patches:
+     - Refresh 01-soname.patch.
+     - Drop 02-pkgconfig.patch (fixed upstream).
+
+ -- Fabian Wolff <fabi.wolff@arcor.de>  Fri, 29 Jan 2021 17:44:09 +0100
+
+z3 (4.8.9-1) unstable; urgency=medium
+
+  * New upstream release.
+  * Upgrade to debhelper compat level 13.
+  * Update patches:
+     - Refresh 00-intrinsics.patch.
+     - Drop 01-reproducibility.patch (fixed upstream).
+     - Refresh 02-soname.patch and rename it to 01-soname.patch.
+     - Drop 03-pthread.patch (fixed upstream).
+     - Add 02-pkgconfig.patch to correct the new z3.pc file.
+  * Add the superficial pkg-config-test autopkgtest to check whether
+    the new z3.pc file is installed properly and usable.
+
+ -- Fabian Wolff <fabi.wolff@arcor.de>  Mon, 21 Sep 2020 12:55:45 +0200
+
+z3 (4.8.8-1) unstable; urgency=medium
+
+  * New upstream release.
+  * Add nojava build profile to make building the Java bindings optional
+    (Closes: #948894).
+  * Remove erroneous Multi-Arch: foreign from libz3-java in debian/control
+    (see the discussion in #948894 for details).
+  * Refresh/update patches.
+  * Upgrade to Standards-Version 4.5.0 (no changes).
+  * Forward patches to upstream.
+
+ -- Fabian Wolff <fabi.wolff@arcor.de>  Sat, 06 Jun 2020 16:32:57 +0200
+
+z3 (4.8.7-4) unstable; urgency=medium
+
+  * Remove all :all and :native annotations from the build dependencies in
+    debian/control (Closes: #948763).
```

Comment:
So is the entire point to ensure builds for Debian work. 
This should be addressable by loading an appropriate Docker container similar to what we do for Centos.
There we use "container: "quay.io/pypa/manylinux2010_x86_64:latest"
What is the debian version? Is it something like "debian:bullseye"? 
My point of reference is: https://hub.docker.com/_/debian

Reply:
nack on this, the idea is to use an Ubuntu container, because debian defaults to zEC12 (and that architecture works)